check_camoufox_type.py

```
@@ -0,0 +1,8 @@
+from camoufox.sync_api import Camoufox
+
+try:
+    with Camoufox(headless=True) as obj:
+        print(f"Yielded object type: {type(obj)}")
+        print(f"Yielded object dir: {dir(obj)}")
+except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)